Description: remove some flag to pass on debian/rules
Author: Nicolas HOUDELOT (nicolas@demosdebs.org)
Last-Update: 2020-04-30
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/configure.ac
+++ b/configure.ac
@@ -28,18 +28,14 @@
 AC_TYPE_UINT32_T
 AC_TYPE_UINT8_T
 
-AC_FUNC_MALLOC
+AC_CHECK_DECLS([malloc])
 AC_CHECK_FUNCS([floor pow sqrt])
 
 AC_CHECK_GL([no])
 AC_CHECK_GTK([2.4.10], [no])
 AC_CHECK_SDL([1.2.4])
 AC_CHECK_SDL_IMAGE([no])
-AC_ADD_CCFLAG([-flto])
-AC_ADD_CCFLAG([-O3])
-AC_ADD_CCFLAG([-ffast-math])
 AC_ADD_CCFLAG([-std=gnu90])
-AC_ADD_LDFLAG([-flto])
 AC_ADD_LIB([-lm])
 
 # Vapaavalintaiset jutut
